### PR TITLE
export fluid-static container schema from fluid-framework

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

**Review by tylerbutler — APPROVED** [review]

**Review by sumedhb1995 — APPROVED** [review]
Thanks!! Looks like the package-lock got changed though, perhaps revert before merging?

**Comment by micahgodbolt** [comment]
yeah, couldn't remember if that was required to be added or not. i can revert

**Comment by micahgodbolt** [comment]
weird, i checked out the package-lock.json from main and committed it....but it still shows a diff here. Locally it says it's the same

**Comment by micahgodbolt** [comment]
Ok new approach that satisfies layer check!

fluid-static moved into the uber package, and client packages changed to import from the uber package. get-container stays put.

**Comment by tylerbutler** [comment]
> Ok new approach that satisfies layer check!
> 
> 
> 
> fluid-static moved into the uber package, and client packages changed to import from the uber package. get-container stays put. 

@micahgodbolt I think this makes a lot of sense! We're making the client packages depend on the "official" API surface that is exposed through the uber package. Cool!

File: tools/build-tools/data/layerInfo.json [hunk 1]
```diff
@@ -152,17 +152,17 @@
...
+                    "@fluid-experimental/get-container"
                 ],
                 "deps": [
                     "Loader",
                     "Routerlicious-Driver",
                     "Server-Libs",
                     "Server-Shared-Utils",
-                    "Test"
+                    "Test",
+                    "Framework"
```
Comment (tylerbutler) [inline tools/build-tools/data/layerInfo.json:165]: nit: alphabetize

**Review by tylerbutler — APPROVED** [review]

File: tools/build-tools/data/layerInfo.json (hunk 1 above)
Comment (micahgodbolt) [reply tools/build-tools/data/layerInfo.json:165]: WAIT FOR PASSING BUILD :D

Comment (tylerbutler) [reply tools/build-tools/data/layerInfo.json:165]: Y U NO COMPILE FIRST? 😋

**Review by tylerbutler — APPROVED** [review]

File: tools/build-tools/data/layerInfo.json (hunk 1 above)
Comment (micahgodbolt) [reply tools/build-tools/data/layerInfo.json:165]: i was able to build just fine, but had some linting and other errors...i just want a full CD pass before making a tweak like that

Comment (micahgodbolt) [reply tools/build-tools/data/layerInfo.json:165]: AND WE GOT IT :D

**Comment by micahgodbolt** [comment]
i'm out for the day. feel free to merge once this passes again

**Pull request merged**